PatchesAndAreasApi/serverless.yml

```diff
@@ -84,6 +84,15 @@ resources:
           - arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole
           - arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess
         Policies:
+          - PolicyName: postToSns
+            PolicyDocument:
+              Version: '2012-10-17'
```

Comment:
This is quite old, is this okay?

Reply:
yep - couldn't find a more updated version